ocaml/src/asd_test.ml

```diff
@@ -71,7 +67,7 @@ let with_asd_client ?(is_restart=false) ?write_blobs test_name port f =
     begin
       capacity := 10_000_000L;
       Unix.system (Printf.sprintf "rm -rf %s" path) |> ignore;
-      Unix.mkdir path 0o777
+      Unix.system (Printf.sprintf "mkdir -p %s" path) |> ignore
```

Comment:
This `-p` option for `mkdir` also creates any missing parent directories, which can be handy on some systems.
In particular this is convenient when I run the tests on my machine, for jenkins it's not needed.